docs/content/preview/develop/quality-of-service/transaction-priority.md

```diff
@@ -157,3 +160,81 @@ yugabyte=> select * from account;
   </tr>
 
 </table>
+
+The following is another example which describes the usage of `yb_transaction_priority` based on the priority types. A transaction is a high priority one if it includes `SELECT`, `UPDATE` operations and so on.
```

Comment:
Let's rewrite the example (instead of copying it from the text I've sent to you).

Specifically:
* A transaction is high priority if, for example, it is a `SELECT ... FOR UPDATE` query.
* Use examples to show that we are picking a random priority.